packages/components/src/components/form/Select/index.tsx

```diff
@@ -32,37 +34,39 @@ const selectStyle = (
         margin: 0,
         padding: '0 8px',
         color: isClean ? theme.TYPE_LIGHT_GREY : theme.TYPE_DARK_GREY,
-        fontSize: variables.NEUE_FONT_SIZE.SMALL,
-        fontWeight: variables.FONT_WEIGHT.MEDIUM,
+        fontSize: NEUE_FONT_SIZE.SMALL,
+        fontWeight: FONT_WEIGHT.MEDIUM,
         borderStyle: 'none',
         justifyContent: isClean ? 'flex-end' : 'flex-start',
+        position: 'static',
```

Comment:
Is these two properties necessary? I think these values are their defaults anyway.

Reply:
I didn't add them, just moved from the Style Component, did not check if they are needed or not. Maybe something is changing the default props and they are needed to be brought back for something...